src/core/ext/transport/inproc/inproc_transport.cc

```diff
@@ -670,6 +678,12 @@ static void op_state_machine(grpc_exec_ctx* exec_ctx, void* arg,
           nullptr);
       s->recv_initial_md_op->payload->recv_initial_metadata
           .recv_initial_metadata->deadline = s->deadline;
+      if (s->recv_initial_md_op->payload->recv_initial_metadata
+              .trailing_metadata_available != nullptr) {
+        *s->recv_initial_md_op->payload->recv_initial_metadata
+            .trailing_metadata_available =
+                other != nullptr && other->send_trailing_md_op != nullptr;
```

Comment:
Can I request a set of parens around the contents of line 685?  Line 683-684 is so long that it is easy to misparse this.

Reply:
Done.